cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -8,7 +8,7 @@
 
 #include "traits.h"
 #include <array>
-#include <basix/finite-element.h>
+#include <basix/basix.h>
```

Comment:
Good practice is to include only what is needed rather than a kitchen sink header.